### ISSUE TITLE
Unclear sentence

### ISSUE THREAD

**Issue by curquiza** [meilisearch/documentation#69]
In [this section](https://docs.meilisearch.com/advanced_guides/synonyms.html#the-multi-way-association), there is this sentence:
```
However, in the case of word to sentence or sentence to sentence
```

I don't understand it, it looks like a sentence that is not finished :)

**Comment by bidoubiwa** [comment]
I made a first attempt at making this less confusing on the `unclear_synonym_sentence` but I think i need some theory on which ranking rule is triggered in the case of a `word => sentence` or `sentence => sentence` @Kerollmops

**Comment by Kerollmops** [comment]
When _one word_ is associated with another _one word_ (i.e. smartphone = iPhone) the two words are exactly considered equal, it means that documents matching even one or the other word are exactly considered equal.

But, if we are in the case where _multiple words_ are associated to a _one/multiple words_  or a _one word_ is associated with _multiple words_, In other words, all other cases than _one to one word_, we do not consider the synonyms at the same level (i.e. SF = San Fransisco). It means that if we find a document with SF it is considered better than a document containing San Fransisco.

**Comment by bidoubiwa** [comment]
fixed: #103
